docs/src/Flex.doc.js

```diff
@@ -13,7 +14,7 @@ card(
   <PageHeader
     name="Flex"
     description={`
-      Flex is a layout component with a very limited subset of the props available to Box and a few special props of its own.
+      Flex is a layout component with a very limited subset of the props available to [Box](/Box) and a few special props of its own.
```

Comment:
I think this is still underselling the usefulness of `gap`, but we can revisit later

Reply:
Yeah since this is the Flex docs, we can address it when we re-write these